common/src/main/java/net/william278/huskclaims/claim/ClaimWorld.java

```diff
@@ -460,6 +460,29 @@ public boolean isBannedFromClaim(@NotNull OnlineUser user, @NotNull Claim claim,
         return false;
     }
 
+    // Check if a claim is private for user
+    @ApiStatus.Internal
+    public boolean CheckIfClaimIsPrivate(@NotNull OnlineUser user, @NotNull Claim claim, @NotNull HuskClaims plugin) {
```

Comment:
Let's clarify this method name a bit - canAccessPrivateClaim perhaps?

Reply:
Indeed, much better :)